src/Illuminate/Notifications/DatabaseNotification.php

```diff
@@ -52,7 +53,7 @@ public function notifiable()
      */
     public function markAsRead()
     {
-        $this->forceFill(['read' => true])->save();
+        $this->forceFill(['read' => Carbon::now()])->save();
```

Comment:
I've never used that before.

Is it done like this:

`$this->forceFill(['read' => $this->freshTimestamp()])->save();`